### PR TITLE
Ufo servlet

### PR DESCRIPTION
**Pull request by niccireinking**

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/UFODataServlet.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+@WebServlet("/ufo-data")
+public class UfoDataServlet extends HttpServlet {
+
+    private JsonArray ufoSightingArray;
+
+    @Override
+    public void init() {
+        ufoSightingArray = new JsonArray();
+        Gson gson = new Gson();
+        Scanner scanner = new Scanner(getServletContext().getResourceAsStream("/WEB-INF/ufo-data.csv"));
```
Comment (Rachelware) [inline src/main/java/com/google/codeu/servlets/UFODataServlet.java:25]: Could you explain how this section only takes in longitude and latitude, not the other information separated by commas? (lines 25-28)

File: src/main/java/com/google/codeu/servlets/UFODataServlet.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+            ufoSightingArray.add(gson.toJsonTree(new UfoSighting(lat, lng)));
+        }
+        scanner.close();
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("application/json");
+        response.getOutputStream().println(ufoSightingArray.toString());
```
Comment (Rachelware) [inline src/main/java/com/google/codeu/servlets/UFODataServlet.java:41]: What does this println do? Where does it print to?

File: src/main/java/com/google/codeu/servlets/UFODataServlet.java (hunk 1 above)
Comment (niccireinking) [reply src/main/java/com/google/codeu/servlets/UFODataServlet.java:25]: Yes, so this code just gets the lat and long coordinates. I had an error and I didnt erase the rest of the data in the CSV file so it just had the lat and long points. I fixed that :)

File: src/main/java/com/google/codeu/servlets/UFODataServlet.java (hunk 2 above)
Comment (niccireinking) [reply src/main/java/com/google/codeu/servlets/UFODataServlet.java:41]: It prints the json.  if you go to localhost:8080/UfoDataServlet.java you will see the json representation of all the lat and long points.

**Review by Rachelware — APPROVED** [review]
lgtm

**Pull request merged**